packages/validator/contracts/test/TestSingleClaimantExecutor.sol

```diff
@@ -0,0 +1,45 @@
+//SPDX-License-Identifier: MIT
```

Comment:
Can you add some tests that use this?

Reply:
Added in `packages/validator/test/SingleClaimantExecutor.test.ts`